### DISCUSSION TITLE
AppImage and/or .deb builds in release "assets"

### DISCUSSION THREAD

**Discussion by eduard5557** [doublecmd/doublecmd#1869] [General]
Hi,

can you please include the AppImage and/or .deb builds
```
doublecmd-gtk-latest-x86_64.AppImage
doublecmd-qt-latest-x86_64.AppImage
```
 in the "assests" for the latest release?

Thank you

**Comment by JonnyTech** [comment]
They are linked in the release notes:

![image](https://github.com/user-attachments/assets/1def9107-13f4-40ce-bace-a213e324a0f7)

Is that not enough?

**Comment by eduard5557** [comment]
I know where I can go , click and download them.

I use a script that is using   [lastversion](https://github.com/dvershinin/lastversion) for several programs hosted on github to check for new version, download and install them.
To work also for doublecmd the AppImage should be listed in assets for the last release
